1_13.via-method.html

```diff
@@ -45,6 +45,7 @@ <h2 class="subtitle">Select which contact details should we <br> use to reset yo
         <div class="authorization">
           <form class="form authorization__form" action="signin" method="post">
             <label class="form__field form__field_blocked">
+              <input class="hidden" type="radio" value="sms">
```

Comment:
Both inputs should contain `name` attribute with single name.

Reply:
done